package.json

```diff
@@ -1,94 +1,68 @@
 {
   "name": "react-web-booster-kit",
-  "version": "1.0.1",
+  "version": "2.0.0",
   "homepage": "boosters.joinspartan.com",
   "description": "Get started with React, Redux, and React-Router!",
   "main": "index.js",
   "private": "true",
   "engines": {
-    "node": ">=7.4.0"
+    "node": ">=8.3.1"
   },
   "scripts": {
-    "analyze": "flow",
-    "build": "bnr build",
+    "build": "yarn run clean && yarn run init && bnr build",
     "clean:project": "yarn cache clean && yarn install --force",
-    "clean": "rimraf dist",
-    "compile": "bnr compile",
-    "debug:dev": "bnr debug:dev",
-    "debug:prod": "bnr debug:prod",
+    "clean": "rimraf build",
```

Comment:
The `clean` and `clean:project` scripts seem to run two completely different sets of commands. It seems like at least one of these should be `clean` and the other be `reset` so that it is more clear.